### ISSUE TITLE
Generated types are prefixed with "pages/"

### ISSUE THREAD

**Issue by OliverDudgeon** [tatethurston/nextjs-routes#76]
My `pages` folder is in the project root. The generated types file looks like this:
```ts
export type Route =
    | { pathname: "pages/_app"; query?: Query | undefined }
    | { pathname: "pages/_document"; query?: Query | undefined }
    | { pathname: "pages/_error"; query?: Query | undefined }
```

hence, I get type errors as the correct paths don't have the `pages/` prefix.

**Comment by tatethurston** [comment]
@OliverDudgeon Sorry about that, thanks for reporting. I revoked the publish of 0.1.1 and published 0.1.2.

**Comment by OliverDudgeon** [comment]
Thanks for the rapid fix!
